### ISSUE TITLE
IMU integration

### ISSUE THREAD

**Issue by igor-nap** [uzh-rpg/rpg_svo#35]
I'm trying to figure out how IMU data could be integrated with the system to improve performance. I see that there are use_imu parameters in config.h and config.cpp but I haven't found instances where the parameter or useImu() is used to get relative rotations.

Is this currently implemented? If so, where and how should the IMU data be streamed into the system?

**Comment by ayberkozgur** [comment]
Hello everyone,
Is IMU integration available at this time?

**Comment by enjoychang** [comment]
Hi @cfo,
I saw in your demo video, you was able control quad with SVO. This cannot be done unless (1) gravity vector in SVO has been aligned with IMU, (2) scale in SVO has been sufficiently recovered. Could you please share a little bit about how you achieved the control in the demo. 
If both factors can be set manually, then how do we do that?

Thank you very much for your time in advance.

Regards,
Chang

**Comment by cfo** [comment]
Hi @enjoychang,
as described in the paper, we use https://github.com/ethz-asl/ethzasl_msf to fuse the output of SVO with an IMU.
best,
christian

**Comment by ginowu** [comment]
Hi @cfo,
 Do you mean this paper: http://rpg.ifi.uzh.ch/docs/RSS15_Forster.pdf ? 

Regards,
Gino

**Comment by highlightz** [comment]
@cfo I am afraid the link https://pixhawk.org/dev/ros/visual_estimation is corrupted. Is there any other relevant description about your work?Tks.

**Comment by AlexisTM** [comment]
@highlightz dev.px4.io is the new information site for the Pixhawk

**Comment by AlexisTM** [comment]
@mhkabir Yes, there is. The closed source version is now available as binaries for noncommercial use.

SVO 2.0 looks really awesome! http://rpg.ifi.uzh.ch/svo2.html
